ui/src/react-admin/modules/content-page/views/ContentEditContentBlocks.tsx

```diff
@@ -69,9 +69,10 @@ const ContentEditContentBlocks: FunctionComponent<ContentEditContentBlocksProps>
 
 	// Methods
 	const handleAddContentBlock = (configType: ContentBlockType) => {
-		const newConfig = CONTENT_BLOCK_CONFIG_MAP[configType](
-			(contentPageInfo.content_blocks || []).length
-		);
+		const length = (contentPageInfo.content_blocks || []).length
+		const newConfig = CONTENT_BLOCK_CONFIG_MAP[configType](length);
+
+		newConfig.id = new Date().valueOf() + length;
```

Comment:
might be best to add a prefix so we can identify these
```suggestion
		newConfig.id = 'NEW_BLOCK_TEMP_ID_' + new Date().valueOf() + '_' + length;
```

Reply:
Needs to be `number`